Nick_Maven/webdriver/Hardcore/POModel/CloudGoogleHomePage.java

```diff
@@ -0,0 +1,38 @@
+package Nick_Maven.webdriver.Hardcore.POModel;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class CloudGoogleHomePage {
+    private WebDriver driver;
+    public static final String HOMEPAGE_URL = "https://cloud.google.com/";
+    public static final int WAIT_TIMEOUT_SECONDS = 10;
+    public static String TEXT_FOR_SEARCH;
+
+
+
+    @FindBy(xpath = "//*[@class='button button-white devsite-suggest-all-results']")
```

Comment:
This is 3 classes:
button
button-white
devsite-suggest-all-results
If all of them need for locator use contains().